GameServer/Logic/Game.cs

```diff
@@ -20,86 +20,76 @@
 using LeagueSandbox.GameServer.Logic.Content;
 using LeagueSandbox.GameServer.Logic.Chatbox;
 using Ninject;
+using LeagueSandbox.GameServer.Logic.Players;
 
 namespace LeagueSandbox.GameServer.Core.Logic
 {
     public class Game
     {
-        private Logger _logger = Program.Kernel.Get<Logger>();
-
         protected Host _server;
-        protected BlowFish Blowfish;
-        protected uint _dwStart = 0x40000000; //new netid
+        protected BlowFish _blowfish;
         protected object _lock = new object();
 
         protected bool _started = false;
         protected int _playersReady = 0;
 
-        protected List<Pair<uint, ClientInfo>> _players = new List<Pair<uint, ClientInfo>>();
         private Map _map;
-        public PacketNotifier PacketNotifier { get; protected set; }
-        public PacketHandlerManager PacketHandlerManager { get; protected set; }
+        public PacketNotifier PacketNotifier;
+        public PacketHandlerManager PacketHandlerManager;
```

Comment:
Yeah, I want to eventually remove these public properties from Game.cs but there is a lot of other work to do before that is _really_ possible.